raster.py

```diff
@@ -365,6 +367,8 @@ def get_predictions_at_each_pixel(
 
       # Surround in try/except as we will be trying to fetch out of bounds data.
       try:
+        if geometry_mask and pd.isnull(geometry_mask.value_at(x, y)):
```

Comment:
need a test case for this in the raster test notebook to ensure that this does not throw an exception.
I think it doesnt, but its worth have a simple test case that runs both.

Reply:
We actually rely on value_at throwing an exception when a coord is out of bounds, much like the logic below it.